src/content/docs/en/guides/styling.mdx

```diff
@@ -514,6 +514,30 @@ You can apply global styles to your Markdown content by adding [`<style>` tags](
 
 If you are using Tailwind, the [typography plugin](https://tailwindcss.com/docs/typography-plugin) can be useful for styling Markdown.
 
+## Production
+
+### Bundle control
+
+When Astro builds your site for production deployment it combines your CSS into chunks. Each page on your site is its own chunk, and CSS that is shared between multiple pages are split off into their own chunks.
+
+In each of your HTML files there will be `<link rel="stylesheet">` tags added for each of the chunks that the pages needs.
+
+In a site with a large number of pages and many different shared pieces of styles this can some times lead to many requests for styles. Astro has a way to minimize requests through the `inlineStylesheets` option. Using this option Astro will put the (minimized) stylesheet into a `<style>` tag rather than request it externally.
+
+```js
+import { defineConfig } from 'astro/config';
+
+export default defineConfig({
+  build: {
+    inlineStylesheets: 'auto'
+  }
+});
+```
+
+When using the `'auto'` option Astro will inline stylesheets that are below the `vite.build.assetsInlineLimit` threshold. This is usually a good option as bigger sheets, such as global ones used by all pages, will be fetched externally and cached, while smaller sheets, such as a page's own styles, will be inlined.
+
+You can also set this option to `'always'` which will inline all stylesheets.
```

Comment:
```suggestion
The `'auto'` option will inline only the stylesheets that are below the `vite.build.assetsInlineLimit` threshold, reducing the number of requests for smaller sheets. Larger stylesheets, such as global ones used by all pages, will still be fetched externally and cached. This option provides a balance between ...

You can also set this option to `'always'` which will inline all stylesheets.
```
Wondering whether a tiny line at the end to make the tradeoff clear might be useful?